### PR TITLE
Move to `org.playframework` for Play 3

### PR DESCRIPTION
**Pull request by mkurz**

### CODE REVIEW

File: documentation/manual/hacking/BuildingFromSource.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Navigate to your existing Play project and make the following edits in `project/
 
 ```scala
 // Change the sbt plugin to use the local Play build (2.8.0-SNAPSHOT)
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.0-SNAPSHOT")
+addSbtPlugin("org.playframework" % "sbt-plugin" % "2.8.0-SNAPSHOT")
```
Comment (ihostage) [inline documentation/manual/hacking/BuildingFromSource.md:78]: How about change version to 3.0.0-SNAPSHOT to be more actual nowadays? 🤔

Comment (mkurz) [reply documentation/manual/hacking/BuildingFromSource.md:78]: :+1: Will push more in one minute

File: documentation/manual/working/scalaGuide/main/ws/ScalaWS.md [hunk 2]
```diff
@@ -178,11 +178,11 @@ The WSResponse extends [`play.api.libs.ws.WSBodyReadables`](api/scala/play/api/l
 
 ### Processing a response as JSON
 
-You can process the response as a [JSON object](https://static.javadoc.io/com.typesafe.play/play-json_2.13/2.9.2/play/api/libs/json/JsValue.html) by calling `response.json`.
+You can process the response as a [JSON object](https://static.javadoc.io/org.playframework/play-json_2.13/2.9.2/play/api/libs/json/JsValue.html) by calling `response.json`.
```
Comment (ihostage) [inline documentation/manual/working/scalaGuide/main/ws/ScalaWS.md:181]: This link (and one below) will never work 🤷‍♂️  We need to release play-json with new groupId.

Comment (mkurz) [reply documentation/manual/working/scalaGuide/main/ws/ScalaWS.md:181]: I know, we need release everything with new groupId, I start with this project, release M1 so other can depend on it, then  continue scalatestplus play, play-json, etc.

Comment (ihostage) [reply documentation/manual/working/scalaGuide/main/ws/ScalaWS.md:181]: Got it 👍

File: documentation/manual/hacking/BuildingFromSource.md (hunk 1 above)
Comment (mkurz) [reply documentation/manual/hacking/BuildingFromSource.md]: Pushed, will merge when green.

**Pull request merged**